src/styles/components/_modal.scss

```diff
@@ -1,88 +1,74 @@
-.bsds-modal
-{
+.bsds-modal {
+  // NOTE: This is a one off size. We can keep an eye out as we build more modal/panel type components to see if this is a dimension we want to flip to a variable.
+  min-width: min(37.5rem, 100vw);
```

Comment:
was this an auto-fix situation?

Reply:
Combo, I moved it since the autofix makes this
```css
  min-width: min(
    37.5rem,
    100vw
  ); // NOTE: This is a one off size. We can keep an eye out as we build more modal/panel type components to see if this is a dimension we want to flip to a variable.
```